src/Libraries/CoreNodeModels/Properties/Resources.resx

```diff
@@ -242,7 +242,7 @@
     <comment>Description for Formula</comment>
   </data>
   <data name="FormulaPortDataResultToolTip" xml:space="preserve">
-    <value>Result of math computation</value>
+    <value>String representation of the object.</value>
```

Comment:
same here

Reply:
Same as the one, I changed descriptions both in Resources.resx and in Resources.en-US.resx

Reply:
Resolved by using appropriate tooltip